webapp/graphite/dashboard/views.py

```diff
@@ -214,7 +214,7 @@ def find(request):
   results = []
 
   # Find all dashboard names that contain each of our query terms as a substring
-  for dashboard in Dashboard.objects.all():
+  for dashboard in sorted(Dashboard.objects.all(), key=lambda dash: dash.name):
```

Comment:
@brutasse I definitely prefer your approach (works fine here) but while the XHR request comes back sorted properly, the UI displays them in [descending order](https://github.com/graphite-project/graphite-web/blob/master/webapp/content/js/dashboard.js#L2827-L2830). Easy fix, I'll open a separate PR for the full change.